metrics/metrics.go

```diff
@@ -1,6 +1,5 @@
 package metrics
 
```

Comment:
does anything actually change in this file?

Reply:
It's just gofmt